platform-ajax/src/main/java/com/softicar/platform/ajax/testing/selenium/engine/level/low/AjaxSeleniumLowLevelTestEngineOutput.java

```diff
@@ -229,4 +239,26 @@ public IDomModalPromptNodes<IDomNode> findModalPromptOrFail() {
 
 		return new DomModalPromptNodes<>(this::findNodeOrFail);
 	}
+
+	// ------------------------------ CSS assert methods ------------------------------ //
+
+	public void assertCssClasses(Collection<ICssClass> expectedClasses, IDomNode node) {
+
+		var expectedValue = expectedClasses//
+			.stream()
+			.map(ICssClass::getName)
+			.sorted()
+			.collect(Collectors.joining(" "));
+		assertEquals(expectedValue, getAttributeValue(node, "class"));
```

Comment:
Can we rely on the stability of the order of the CSS classes, as returned by `getAttributeValue(node, "class")`? E.g. are they always ordered alphabetically?
If not, we should probably deal with that to avoid random failures... for instance, we could create two sets of CSS class names, and compare those sets.

Reply:
no they are always sorted alphabetically but `DomElementCssClassAttributeManager`